nyaa/models.py

```diff
@@ -771,6 +771,44 @@ def __init__(self, info_hash, method):
         self.method = method
 
 
+class RangeBan(db.Model):
+    __tablename__ = 'rangebans'
+
+    id = db.Column(db.Integer, primary_key=True)
+    _cidr_string = db.Column(db.String(length=18), nullable=False)
+    masked_cidr = db.Column(db.BigInteger, nullable=False,
+                            index=True)
+    mask = db.Column(db.BigInteger, nullable=False, index=True)
+    enabled = db.Column(db.Boolean, nullable=False, default=True)
+    # If this rangeban may be automatically cleared once it becomes
+    # out of date, set this field to True.
+    temp = db.Column(db.Boolean, nullable=False, default=False)
```

Comment:
It might be useful to have a creation date if we want to do actual expiry instead of a static cronjob removing all temporary bans (which means that one ban added fifteen minutes ago is cleared).

Also this could just be `temporary`.